senders/mattermost/sender.go

```diff
@@ -187,15 +199,45 @@ func (sender *Sender) buildEventsString(events moira.NotificationEvents, charsFo
 	return eventsString
 }
 
-func (sender *Sender) sendMessage(message string, contact string, triggerID string) error {
+func (sender *Sender) sendMessage(message string, contact string, triggerID string) (string, error) {
```

Comment:
So that the meanong of the returned value would be obvious from it's type 

```suggestion
func (sender *Sender) sendMessage(message string, contact string, triggerID string) (*model.Post, error) {
```